azurelinuxagent/common/osutil/default.py

```diff
@@ -79,6 +79,10 @@ def _get_firewall_accept_command(wait, command, destination, owner_uid):
     return AddFirewallRules.get_iptables_accept_command(wait, command, destination, owner_uid)
 
 
+def _get_firewall_accept_command_nonroot_tcp(wait, command, destination):
+    return AddFirewallRules.get_iptables_accept_command_nonroot_tcp(wait, command, destination)
```

Comment:
Can you post a comment on how the iptable command would look for this here please?

Reply:
Done